llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1261,6 +1261,9 @@ LogicalResult TopkOp::verify() {
     return op->emitOpError("incompatible input/output shapes");
   }
   // Check region compatibility
+  if (!region().getBlocks().size()) {
```

Comment:
This check should be done in Tablegen. Adding `SingleBlockImplicitTerminator<"YieldOp">` must do the trick ( I might have the syntax wrong.